docs/developer/typescript/01_gettingStarted/01_hello/01_ecs.md

```diff
@@ -0,0 +1,61 @@
+# The ECS Pattern
+The [Entity Component System](https://en.wikipedia.org/wiki/Entity_component_system) is a pattern used to organize our code when writing software.  
+In this pattern:
+- Logic is represented as `System`s, and they define the behavior of the application.
+- Data is represented as `Component`s that have no behavior or identifiers attached to them.
+- Components are attached to `Entities`.
+- An `Entity` is essentially a "name" that can be used to connect a group of components into a single "thing".
```

Comment:
I think the Entity description is a bit rough. I think the ECS page on wikipedia does it justice 
`An entity represents a general-purpose object. In a game engine context, for example, every coarse game object is represented as an entity. Usually, it only consists of a unique id.`(For EE the unique id is an integer)

Reply:
This is trying to give the `Simple English` version of what an entity is, just like the other two explanations do. Simplification sometimes has the cost of slight inaccuracies in meaning.
But will give it some thought to see if it can be reworded bit so that its both simple and more correct.

Reply:
> An entity represents a general-purpose object. In a game engine context, for example, every coarse game object is represented as an entity. Usually, it only consists of a unique id
I found the issue:
- The keyword in that definition is `represents`
- The keyword in my definition is `is`

The wikipedia definition is describing what an entity represents, and gives less emphasis what it is. It only uses that single short sentence at the end to explain what an entity truly is.

And I'm explaining _only_ what an entity `is`, using very simple terms.

Rewording the wikipedia definition to fit the same goal as mine would result in:
> An entity [... is] a unique id

I'm currently rewording mine to represent this difference, but you could see my definition as an attempt at wording that very last sentence with more intutive and `Simple English` wording.

Reply:
You can **not** simplify language and change the meaning. It's like compressing an image but losing to much data. The explination is not correct and distracting. `An Entity is an "object" that you can assign components to. Typically implemented with a uniqueId. i.e. A game object such as the skybox or a chair. A component such as their Transform data or Material.` These are important things the creator will need to succeed. Our current target for the creator is a prosumer. Examples: Indie app/3D/game developer, Game development studio.

Reply:
Just pushed https://github.com/EtherealEngine/etherealengine-docs/pull/114/commits/9b52595d6f41cf80fd97ddfbb0950aa7e3c39492
Please review that